examples/controlnet/train_controlnet.py

```diff
@@ -543,16 +543,13 @@ def make_train_dataset(args, tokenizer, accelerator):
             cache_dir=args.cache_dir,
         )
     else:
-        data_files = {}
         if args.train_data_dir is not None:
-            data_files["train"] = os.path.join(args.train_data_dir, "**")
-        dataset = load_dataset(
-            "imagefolder",
-            data_files=data_files,
-            cache_dir=args.cache_dir,
-        )
+            dataset = load_dataset(
```

Comment:
i downloaded the fusing/fill50k dataset from hugging face to my server.but when i use this scripts, i met an error says that str has no attribute 'convert'.and i found that the examples[image_column] is a string which is different from the examples[image_column] when i using remote hugging face. what happened? how can i use load_dataset to load my local dataset(the files is completely the same as the hugging face remote repository)

Reply:
@zhangvia 
use `load_dataset` from a local folder requires a data-loading script (see https://huggingface.co/docs/datasets/dataset_script)

If you just want to use fill50k dataset, just load from the hub - I think it will only download once and use the cache if it can find one

Reply:
hey @zhangvia I'm not sure I follow your bug. It might be that your dataloading script does not return the image column as a PIL image. See https://huggingface.co/datasets/fusing/fill50k/blob/main/fill50k.py for an example of how to do so

Reply:
> hey @zhangvia I'm not sure I follow your bug. It might be that your dataloading script does not return the image column as a PIL image. See https://huggingface.co/datasets/fusing/fill50k/blob/main/fill50k.py for an example of how to do so

actually，I didn't write a script by myself.I just use the fill50k.py. but i found that the reason why the script didn't work....because my folder name is not the same as the script.....